velox/functions/sparksql/tests/BitwiseTest.cpp

```diff
@@ -42,6 +42,30 @@ class BitwiseTest : public SparkFunctionBaseTest {
   std::optional<T> bitwiseOr(std::optional<T> a, std::optional<T> b) {
     return evaluateOnce<T>("bitwise_or(c0, c1)", a, b);
   }
+
+  template <typename T>
+  std::optional<T> shiftLeft(std::optional<T> a, std::optional<T> b) {
```

Comment:
nit: Since you arent passing empty any reason to use std::optional?

Reply:
Just follow https://github.com/facebookincubator/velox/blob/d74bd95866e6e5a1a203bbcef4b473c991213465/velox/functions/prestosql/tests/BitwiseTest.cpp#L80-L91.

Thanks!